src/ripple/app/paths/FindPaths.h

```diff
@@ -40,15 +41,15 @@ class FindPaths
         unsigned int const maxPaths);
     ~FindPaths();
 
-    bool findPathsForIssue (
+    boost::optional<std::pair<
+        STPathSet, std::vector<STAmount>>>
+    findPathsForIssue (
         Issue const& issue,
 
-        /** On input, pathsInOut contains any paths you want to ensure are
+        /** paths contains any paths you want to ensure are
             included if still good.
-
```

Comment:
I don't know if there's a need for a struct, but the information about the return value could be added as a function comment. Eg.
```
/**
  Computes paths, blah blah blah.

  @return optional pair. "first" contains valid paths passed in plus and additional paths found. 
  Only non-default paths without source or destination will be added. "second" contains 
  blah blah blah.
*/
```